src/sentry/web/frontend/accounts.py

```diff
@@ -74,27 +74,42 @@ def expired(request, user):
 def recover(request):
     from sentry.app import ratelimiter
 
+    extra = {'ip_address': request.META['REMOTE_ADDR']}
+    if request.META.get('HTTP_USER_AGENT'):
+        extra['user_agent'] = request.META['HTTP_USER_AGENT']
```

Comment:
Can also just do this:

```python
extra = {
  'ip_address': request.META['REMOTE_ADDR'],
  'user_agent': request.META.get('HTTP_USER_AGENT')
}
```

Since if User-Agent is missing, it'll spit back `None`, which our logger would filter out anyways.